src/test/kotlin/no/skatteetaten/aurora/gobo/testDataBuilders.kt

```diff
@@ -297,6 +297,99 @@ data class ApplicationDeploymentDetailsBuilder(
         }
 }
 
+data class ApplicationDeploymentDetailsResourceBuilder(
+    val resourceLinks: Links = Links(),
+    val pause: Boolean = false
+) {
+
+    fun build() =
+        ApplicationDeploymentDetailsResource(
+            updatedBy = "linus",
+            buildTime = Instant.now(),
+            gitInfo = GitInfoResource("123abc", Instant.now()),
+            imageDetails = ImageDetailsResource(
+                imageBuildTime = defaultInstant,
+                dockerImageReference = "docker.registry/group/name@sha256:123",
+                dockerImageTagReference = "docker.registry/group/name:2"
+            ),
+            deployDetails = DeployDetailsResource(
+                targetReplicas = 1,
+                availableReplicas = 1,
+                deployment = "deployment-1",
+                phase = "Complete",
+                deployTag = "foobar",
+                paused = pause
+            ),
+            databases = listOf("123"),
+            podResources = listOf(
+                PodResourceResource(
+                    name = "name",
+                    phase = "status",
+                    deployTag = "tag",
+                    latestDeployTag = true,
+                    replicaName = "deployment-1",
+                    latestReplicaName = true,
+                    containers = listOf(
+                        ContainerResource(
```

Comment:
Kanskje hatt en egen builder for ContainerResource og ManagementResponsesResource? siden de går igjen så mye

Reply:
Laget egen builder  for ContainerResource (det førte til litt endringer i expected values i noen tester).